### DISCUSSION TITLE
forcing model.transcribe to use a particular language

### DISCUSSION THREAD

**Discussion by nairajay2k** [openai/whisper#2088] [Q&A]
import whisper
model = whisper.load_model("base")
result = model.transcribe("someaudiofile.wav",decode_options={'language':'hi'})

Trying to force the transcription to Hindi (hi) but I get 
TypeError: DecodingOptions.__init__() got an unexpected keyword argument 'decode_options'

Pl let me know what I need to do.

Thanks in advance

**Comment by nairajay2k** [comment]
Ok so I think when I use
model.transcribe("someaudiofile.wav",language='hi'}) it works for small but not base model. For base model it returns an urdu/persian text.

**Comment by ulatekh** [comment]
Did you try `options = whisper.DecodingOptions(language = 'hi')` followed by `result = model.transcribe("someaudiofile.wav",decode_options=options)`?
